inception-project/inception
# Feature 919/Add button to leave a project

**What's in the PR**
Add a button to the project overview page allowing to leave a project. Ask before actually leaving since re-joining won't be possible without the project manager's involvement.
Leaving the project is not possible for managers/admins to avoid orphaned projects.

**How to test manually**
* In a role that is not the project's manager/admin: Go to projects overview page and press leave Project button
* Should open a window to confirm your choice and then hide this project from you

**Automatic testing**

**Documentation**
